src/sentry/models/externalissue.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import, print_function
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import BoundedPositiveIntegerField, Model, sane_repr
+
+
+class ExternalIssue(Model):
+    __core__ = False
+
+    organization_id = BoundedPositiveIntegerField(db_index=True)
```

Comment:
Do you need indexes for these individual fields? Since we have the UNIQUE that covers all 3 of them. I don't see you leveraging a query that only uses say, integration_id without organization_id.